docs/build/guides/transactions/create-account.mdx

```diff
@@ -161,39 +169,128 @@ import java.net.*;
 import java.io.*;
 import java.util.*;
 
-String friendbotUrl = String.format(
-  "https://friendbot.stellar.org/?addr=%s",
-  pair.getAccountId());
-InputStream response = new URL(friendbotUrl).openStream();
-String body = new Scanner(response, "UTF-8").useDelimiter("\\A").next();
-System.out.println("SUCCESS! You have a new account :)\n" + body);
+private static final HttpClient HTTP = HttpClient.newBuilder()
+  .connectTimeout(Duration.ofSeconds(20))
+  .build();
+private static final SorobanServer SERVER = new SorobanServer(RPC_URL);
+
+public static void main(String[] args) throws Exception {
+  KeyPair parent = KeyPair.random();
+  log("Secret: %s", new String(parent.getSecretSeed()));
+  log("Public: %s", parent.getAccountId());
+
+  String friendbotResponse = requestAirdrop(parent.getAccountId());
+  log("SUCCESS! You have a new account:\n%s", friendbotResponse);
+
+  Account parentAccount = SERVER.loadAccount(parent.getAccountId());
+
+  KeyPair child = KeyPair.random();
+  AssetTypeCreditAlphaNum12 usdcAsset = Asset.createNonNative("USDC", TESTNET_USDC_ISSUER);
+
+  Transaction transaction = new TransactionBuilder(parentAccount, Network.TESTNET)
+    .setBaseFee(Transaction.MIN_BASE_FEE)
+    .setTimeout(TransactionBuilder.TIMEOUT_INFINITE)
+    .addOperation(new CreateAccountOperation.Builder(child.getAccountId(), "5").build())
+    .addOperation(new ChangeTrustOperation.Builder(usdcAsset, TESTNET_USDC_ISSUER).build())
+    .build();
+  transaction.sign(parent);
+
+  SendTransactionResponse sendResp = sendTransaction(transaction);
+  GetTransactionResponse txResp = SERVER.pollTransaction(sendResp.getHash());
+  long ledgerSeq = txResp.getLedger() != null ? txResp.getLedger() : txResp.getLatestLedger();
+  log("Transaction confirmed in ledger %d", ledgerSeq);
+  log("Created the new account %s", child.getAccountId());
+}
+
+private static String requestAirdrop(String accountId) throws IOException, InterruptedException {
+  HttpRequest request = HttpRequest.newBuilder()
+    .uri(URI.create(FRIENDBOT_URL + accountId))
```

Comment:
does the java run, I don't see where FRIENDBOT_URL is defined, but could be missing something? This could get freindbot url dynamically from `rpc.getNetwork` instead?  did a similar thing in [send/recv](https://github.com/stellar/stellar-docs/pull/1788/files#diff-c72f39490aa000a7c8c92f4596cfe6edaf75a8d0ce55156a85239b05341e96ccR259)